frontend/www/js/omegaup/components/notification/Clarifications.vue

```diff
@@ -0,0 +1,211 @@
+<template>
+  <li class="dropdown">
+    <a
+      aria-expanded="false"
+      aria-haspopup="true"
+      class="notification-button dropdown-toggle"
+      data-toggle="dropdown"
+      href="#"
+      role="button"
+      v-on:click="unread = false"
+      ><span class="glyphicon glyphicon-bell"></span>
+      <span
+        class="notification-counter label"
+        v-bind:class="{ 'label-danger': unread }"
+        v-if="clarifications &amp;&amp; clarifications.length &gt; 0"
```

Comment:
```suggestion
        v-if="clarifications && clarifications.length > 0"
```

el linter ya nos permite usar los caracteres correctos!

Reply:
Oh, perfecto.